build_tools/pkgci/build_linux_packages.sh

```diff
@@ -188,12 +188,13 @@ function build_wheel() {
 function build_iree_runtime() {
   # We install the needed build deps below for the tools.
   IREE_RUNTIME_BUILD_TRACY=ON IREE_RUNTIME_BUILD_TRACY_TOOLS=ON \
-  IREE_EXTERNAL_HAL_DRIVERS="rocm" \
+  IREE_EXTERNAL_HAL_DRIVERS="rocm" IREE_HAL_DRIVER_CUDA=ON \
```

Comment:
As written this will have a merge conflict with https://github.com/iree-org/iree/pull/17029. Should put each option on its own line, and this should also include `IREE_HAL_DRIVER_HIP=ON`